client/ui/controllers/settingsController.cpp

```diff
@@ -27,6 +28,7 @@ SettingsController::SettingsController(const QSharedPointer<ServersModel> &serve
       m_settings(settings)
 {
     m_appVersion = QString("%1 (%2, %3)").arg(QString(APP_VERSION), __DATE__, GIT_COMMIT_HASH);
+    startLoggingWather();
```

Comment:
Please rename function to something like checkIfNeedDisableLogs

Reply:
Fixed)